src/psyclone/psyir/symbols/symbol.py

```diff
@@ -95,23 +95,12 @@ class ImportInterface(SymbolInterface):
     :type container_symbol: \
         :py:class:`psyclone.psyir.symbols.ContainerSymbol`
 
-    :raise TypeError: if the container_symbol is not a ContainerSymbol.
 
     '''
     def __init__(self, container_symbol):
```

Comment:
Since we're here, please could you remove the `object` inheritance from `Symbol` and also modernise the `super()` below.

Reply:
Hopefully you won't see the object after I bring this up to master and I updated the super.